### PR TITLE
Re-enable haptic using hfd-service

### PR DESCRIPTION
**Pull request by tari01**
fixes https://github.com/AyatanaIndicators/ayatana-indicator-datetime/issues/67

### CODE REVIEW

**Comment by sunweaver** [comment]
@mariogrip can you take a look at the usensord -> hfd port in ayatana-indicator-datetime? Thanks.

**Review by sunweaver — APPROVED** [review]
Looks good for me. But @mariogrip should give his 5¢ on this.

File: src/haptic.cpp [hunk 1]
```diff
@@ -93,66 +93,51 @@ class Haptic::Impl
         {
             case MODE_PULSE: // the only mode currently supported... :)
```
Comment (mariogrip) [inline src/haptic.cpp:94]: I guess we dont need this switch anymore, since we dont do anything in here.

**Review by mariogrip — CHANGES_REQUESTED** [review]
Small nitpick, otherwise it looks good

File: src/haptic.cpp (hunk 1 above)
Comment (tari01) [reply src/haptic.cpp:94]: I sooooo wanted to drop this, now I finally can.

**Review by mariogrip — APPROVED** [review]
LGTM :+1:

**Pull request merged**